### PR TITLE
Fix staff users not viewing all courses in studio issue

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:**  This PR adds a fix for staff users not viewing all courses in studio issue

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-3867

**Merge checklist:**

- [ ] All reviewers approved
- [ ] Commits are (reasonably) squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable change.

### CODE REVIEW

**Review by taimoor-ahmed-1 — COMMENTED** [review]
Kindly add/update tests.

**Comment by asadmanzoor93** [comment]
> Kindly add/update tests.

Done

**Review by taimoor-ahmed-1 — APPROVED** [review]
👍

File: openedx/features/edly/tests/test_utils.py [hunk 1]
```diff
@@ -345,16 +345,23 @@ def test_set_global_course_creator_status(self):
         response = cookies_api.set_logged_in_edly_cookies(self.request, HttpResponse(), self.user, cookie_settings(self.request))
         self._copy_cookies_to_request(response, self.request)
         edly_user_info_cookie = self.request.COOKIES.get(settings.EDLY_USER_INFO_COOKIE_NAME)
-        edly_sub_org = get_edly_sub_org_from_cookie(edly_user_info_cookie)
+        edx_orgs = get_edx_org_from_cookie(edly_user_info_cookie)
+
```
Comment (burhandodhy) [inline openedx/features/edly/tests/test_utils.py:349]: nit: remove emtpy line

**Review by burhandodhy — APPROVED** [review]
LGTM

**Pull request merged**